server/store/store.go

```diff
@@ -11,11 +13,12 @@ type Store interface {
 // PollStore allows the access polls in the store.
 type PollStore interface {
 	Get(id string) (*poll.Poll, error)
-	Save(poll *poll.Poll) error
-	Delete(poll *poll.Poll) error
+	Insert(*poll.Poll) error
+	Update(prev *poll.Poll, new *poll.Poll) error
+	Delete(*poll.Poll) error
 }
 
-// SystemStore allows to access system informations in the store.
+// SystemStore allows to access system information in the store.
 type SystemStore interface {
 	GetVersion() (string, error)
 	SaveVersion(version string) error
```

Comment:
I didn't touch these two as they are only run once on a node and multi node coordination would be needed.